modules/10-basics/25-evaluation-order/description.en.yml

```diff
@@ -1,26 +1,26 @@
 ---
 
-name: Порядок вычисления
+name: Evaluation order
 theory: |
-  Попробуем превратить выражение `5 - 3 + 1` в программу на Clojure. С точки зрения арифметики, порядок вычисления элементов этого составного выражения строго определен. Сначала вычисляется `5 - 3`, затем к получившемуся результату прибавляется единица.
+  Let's try to turn the expression `5 - 3 + 1` into a Clojure program. From the point of view of arithmetic, the order of calculating the elements of this compound expression is strictly defined. First, `5 - 3` is calculated, then one is added to the result.
```

Comment:
From the point of view of arithmetic, the order of calculating the elements of this compound expression is strictly defined => The order in which the elements of this compound expression are calculated is strictly determined in terms of arithmetic.